Libraries/dotNetRDF/Core/URINode.cs

```diff
@@ -41,7 +42,8 @@ namespace VDS.RDF
 #if !NETCORE
     [Serializable,XmlRoot(ElementName="uri")]
 #endif
-    public abstract class BaseUriNode 
+    [DebuggerDisplay("{" + nameof(_uri) + "}")]
```

Comment:
Slightly unrelated. Can revert if you prefer

Reply:
Fine with me!